client/yarn.lock

```diff
@@ -3287,6 +3287,11 @@ dotenv@6.2.0:
   resolved "https://registry.yarnpkg.com/dotenv/-/dotenv-6.2.0.tgz#941c0410535d942c8becf28d3f357dbd9d476064"
   integrity sha512-HygQCKUBSFl8wKQZBSemMywRWcEDNidvNbjGVyZu3nbZ8qq9ubiPoGLMdRDpfSrpkkm9BXYFkpKxxFX38o/76w==
 
+dotenv@^7.0.0:
```

Comment:
Create-react-app comes with the dotenv package in react-scripts, we don't need to add it to dependencies explicitly.

Reply:
Fixed.